staging/src/k8s.io/kubectl/pkg/cmd/get/get.go

```diff
@@ -593,7 +593,7 @@ func (o *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []string) e
 	if trackingWriter.Written == 0 && !o.IgnoreNotFound && len(allErrs) == 0 {
 		// if we wrote no output, and had no errors, and are not ignoring NotFound, be sure we output something
 		if allResourcesNamespaced {
-			fmt.Fprintf(o.ErrOut, "No resources found in %s namespace.\n", o.Namespace)
+			fmt.Fprintf(o.ErrOut, "Warning: Multiple resources found in %s namespace with the given name.\nUse different alias or the fully qualified resource name.", o.Namespace)
```

Comment:
This isn't good approach, you're assuming there are multiple resources with the same name but that is not always true. In cases like yours it's is advised always to use full-qualified name to get the requested resource. The list of resources can always be verified using `kubectl api-resources` command.
/hold